src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -96,25 +97,70 @@ public static AminoAcid getEukaryoticAminoAcidByCodon(final String codon) {
     }
 
     /**
-     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code codon}.
+     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code rawCodon}.
      * The codons given are expected to be valid for Mitochondrial DNA.
-     * @param codon The three-letter codon (each letter one of A,T,G,C) representing a Mitochondrial {@link AminoAcid}
-     * @return The {@link AminoAcid} corresponding to the given {@code codon}.  Returns {@code null} if the given {@code codon} does not code for a Mitochondrial {@link AminoAcid}.
+     * Assumes no special cases for alternate initiation sites.
+     * @param rawCodon The three-letter codon (each letter one of A,[T or U],G,C) representing a Mitochondrial {@link AminoAcid}
+     * @return The {@link AminoAcid} corresponding to the given {@code rawCodon}.  Returns {@code null} if the given {@code rawCodon} does not code for a Mitochondrial {@link AminoAcid}.
      */
-    public static AminoAcid getMitochondrialAminoAcidByCodon(final String codon, final boolean isFirst) {
+    public static AminoAcid getMitochondrialAminoAcidByCodon(final String rawCodon) {
+        return getMitochondrialAminoAcidByCodon(rawCodon, false, false, false, false, false);
+    }
 
-        if (codon == null) {
+    /**
+     * Returns the {@link AminoAcid} corresponding to the given three-letter Mitochondrial {@code rawCodon}.
+     * The codons given are expected to be valid for Mitochondrial DNA.
+     * @param rawCodon The three-letter codon (each letter one of A,[T or U],G,C) representing a Mitochondrial {@link AminoAcid}
+     * @param isGenusBosAndFirst {@code True} iff the codon to be decoded is from a sample in the genus Bos (cow) and came first in the coding sequence.
+     * @param isGenusHomoAndFirst {@code True} iff  the codon to be decoded is from a sample in the genus Homo (human) and came first in the coding sequence.
+     * @param isGenusMusAndFirst {@code True} iff  the codon to be decoded is from a sample in the genus Mus (mouse) and came first in the coding sequence.
+     * @param isGenusCorturnixAndFirst {@code True} iff  the codon to be decoded is from a sample in the genus Corturnix (quail) and came first in the coding sequence.
+     * @param isGenusGallusAndFirst {@code True} iff  the codon to be decoded is from a sample in the genus Gallus (chicken) and came first in the coding sequence.
+     * @return The {@link AminoAcid} corresponding to the given {@code rawCodon}.  Returns {@code null} if the given {@code rawCodon} does not code for a Mitochondrial {@link AminoAcid}.
+     */
+    public static AminoAcid getMitochondrialAminoAcidByCodon(final String rawCodon,
+                                                             final boolean isGenusBosAndFirst,
```

Comment:
Several comments here:

Use an enum and a boolean combination rather than this `isGenus...AndFirst` parameter list. 

Is this method called anywhere outside of this class?  I could not find an example.

What do we do if none of the genus match our sample?  How is the genus even being determined?  Wouldn't we want to use the reference?

Reply:
I can simplify it, but the map will take a Pair<GenusEnum, CodonString>, which will be a little gross.  I want to make sure that this mapping is explicitly for those specific genuses.

No - it's not called anywhere else.

I did this because it wasn't clear that we wouldn't want to make this happen in the future.  I added `getMitochondrialAminoAcidByCodon` because of my conversation with Sarah Calvo.  She told me the following when I asked about special overrides for codons:

"I would just code the vertebrate mito codon as the standard codon table with the 4 differences:

AGA | Ter * | Arg R
-- | -- | --
AGG | Ter * | Arg R
AUA | Met M | Ile I
UGA | Trp W | Ter *

So, AUA and AUG code for Methionine (and AUU should still code for Isoleusine)."

That said, it is conceivable that someone would want to call the specific changes by genus, so I left it in as an option to the user.